src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String SPLIT_SYMBOL = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final int TITLE_LINE = 1;
```

Comment:
Constants such as 'SPLIT_SYMBOL', 'OPERATION_INDEX', 'FRUIT_NAME_INDEX', 'QUANTITY_INDEX', and 'TITLE_LINE' should be private if they are not intended to be used outside of this class. This will adhere to the encapsulation principle and prevent unwanted access to these constants.